panelapp/models.py

```diff
@@ -13,13 +13,13 @@ class PaLocusList(models.Model):
         primary_key=True,
     )
 
-    panel_app_id = models.IntegerField(null=False, blank=False, unique=True)
+    url = models.TextField(max_length=2000, null=False, blank=False, unique=True)
+    panel_app_id = models.IntegerField(null=False, blank=False)
```

Comment:
The `URL` has to incorporate `panel_app_id` to become unique.  How about using unique_together?

Reply:
The URL includes the ID, its the full path